src/runtime/basic-runtime.js

```diff
@@ -463,6 +470,29 @@ export class ConfiguredBasicRuntime {
       }
     );
   }
+
+  /**
+   * Handler function.
+   * @param {!../components/activities.ActivityPortDef} port
+   */
+  entitlementsResponseHandler(port) {
```

Comment:
Could you add tests covering this method?

Reply:
Done